app/controllers/etds_controller.rb

```diff
@@ -160,18 +138,12 @@ def submission_attributes # rubocop:disable Metrics/AbcSize, Metrics/MethodLengt
   end
 
   def valid_xml?
-    true if etd_params && dissertation_id && readers
+    etd_params && dissertation_id && readers
```

Comment:
It seemed odd to me that this method could return `false` and `nil`, and I don't think that's a useful distinction. Now it always returns a boolean.